dev/com.ibm.ws.logstash.collector_fat/fat/src/com/ibm/ws/logstash/collector/tests/LogstashSSLTest.java

```diff
@@ -17,6 +17,12 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
+import org.junit.ClassRule;
```

Comment:
I would undo all of the changes here and add in a audit-2.0 server xml and duplicate the testLogstashForAuditEvent since that is the only test that actually does audit-1.0.  No reason to run repeat the other tests since they don't use audit.

Reply:
Done, thanks for the suggestion.